state.go

```diff
@@ -42,14 +42,18 @@ type State struct {
 	// Sending header key and next header key. Only used for header encryption.
 	HKs, NHKs Key
 
-	// Number of ratchet steps after which all skipped message keys for that key will be deleted.
+	// How long we keep messages keys, counted in number of messages received.
+	// For example if MaxKeep is 5 we only keep the last 5 messages keys, deleting everything n - 5
 	MaxKeep uint
 
+	// Max number of message keys per session, older keys will be deleted in FIFO fashion
```

Comment:
nitpick: just to be consistent we should have all comments as full sentences ending with a dot.